.build/parent-pom-template.xml

```diff
@@ -233,12 +233,52 @@
       <id>zznate</id>
       <name>Nate McCall</name>
     </developer>
+    <developer>
+      <id>smiklosovic</id>
+      <name>Stefan Miklosovic</name>
+    </developer>
   </developers>
   <scm>
     <connection>scm:https://gitbox.apache.org/repos/asf/cassandra.git</connection>
     <developerConnection>scm:https://gitbox.apache.org/repos/asf/cassandra.git</developerConnection>
     <url>https://gitbox.apache.org/repos/asf?p=cassandra.git;a=tree</url>
   </scm>
+
+  <profiles>
+    <profile>
+      <id>x86_64</id>
+      <activation>
+        <os>
+          <arch>!aarch64</arch>
```

Comment:
Should we also add `<family>unix</family>'? This will look more correct even if we never run on other operating systems.
Is the condition correct? Should we place here `x86_64` to avoid matching other types of `x86`?

Reply:
What other types, like 32 bits? Does Cassandra even run on 32 bits anyway? I would just keep it like this.

If we run this like ... on FreeBSD, it will put x86 lib on the class path, then it will try to load it and it will presumably fail and log will be printed.

I do not think anybody will ever build Cassandra with Maven on FreeBSD or any other unix for that matter. Like ...why? People are going to at most run it (if even that) and they are covered already as they will have it in `lib/x86_64` or `lib/aarch64`. So them running Cassandra on these platforms will not have any library added to the class path and again this is catched by the exception handling we have in place for the provider.

Reply:
My concern is about putting something in the classpath that doesn't match the architecture type, so even if the cassandra is able to handle this case it's good, but the other applications may not. 
If you think this will work, let's resolve this comment, I just wanted to point out a potential problem, but in fact, it may not even exist :-)

Reply:
for now we need something as a default even if it doesn't successfully load the .so files. 

that's why it's `! aarch64` (as confusing as that might first read.)

let's add a comment explaining the contortion…